src/cmd-build

```diff
@@ -340,7 +340,7 @@ first=true
      checksum=$(sha256sum "${path}" | cut -f 1 -d ' ')
      size=$(stat -c '%s' "${path}")
      if ${first}; then first=false; else echo ','; fi
-     echo '"'"${image}"'": {"path": "'"${path}"'", "sha256": "'"${checksum}"'", "size": "'"${size}"'"}'
+     echo '"'"${image}"'": {"path": "'"${path}"'", "sha256": "'"${checksum}"'", "uncompressed-size": "'"${size}"'"}'
```

Comment:
Hmm, WDYT here to instead keep this as `size`, but teach `cmd-compress` to migrate it to `uncompressed-size` and update `size` to refer to the compressed artifact? I think this matches what we do with the SHA256 too, right?

Reply:
Yeah, you are correct; I'll update to follow that paradigm